### ISSUE TITLE
[advancedmenu] Option to use the original question message for the response

### ISSUE THREAD

**Issue by Flairseds** [sebkuip/mm-plugins#5]
add a new command to delete the old message or edit the old message that lets you choose the menu, like the reply but just edit the msg :)

**Comment by martinbndr** [comment]
Yes nice Idea. Just make it as a config option through ``?advancedmenu config``.
So a toggle option to delete the message after finishing selections.
